cli/tests/pcluster/templates/test_shared_storage.py

```diff
@@ -202,6 +203,21 @@ def assert_sg_rule(
     assert_that(sg_rules).is_length(1)
 
 
+def test_non_happy_ontap_and_openfsx_mounting(mocker, test_datadir):
+    dummy_api = _DummyAWSApi()
+    dummy_api._fsx.set_non_happy(True)
+    mocker.patch("pcluster.aws.aws_api.AWSApi.instance", return_value=dummy_api)
+    mocker.patch("pcluster.aws.ec2.Ec2Client.get_instance_type_info", side_effect=_DummyInstanceTypeInfo)
```

Comment:
is this patching required for this unit test?
If not, let's remove this line.

Reply:
Yes, it is necessary as the Ec2Client expects to have instance_type_info. Otherwise, the test fails.

Reply:
Thank you for the clarification.